src/certificate.c

```diff
@@ -93,6 +96,36 @@ private_key_dealloc (private_key *self)
 	Py_TYPE(self)->tp_free ((PyObject *) self);
 }
 
+static PyObject *
+raise_certificate_loading_error (const char *path, const char *pem)
+{
+	unsigned long e = ERR_peek_error ();
+	char buf[4096];
+	PyObject *args = NULL;
+	PyObject *py_path = NULL;
+	PyObject *py_pem = NULL;
+
+	args = PyTuple_New (3);
+	snprintf (buf, sizeof buf, "%s: %s", ERR_lib_error_string (e), ERR_reason_error_string (e));
```

Comment:
Maybe it would be better to use tuple of four strings and use first string for `ERR_lib_error_string (e)` and seconds for `ERR_reason_error_string (e)`.